### DISCUSSION TITLE
create-next-app documentation is missing?

### DISCUSSION THREAD

**Discussion by MathiasKandelborg** [vercel/next.js#16397] [Help] [answered]
Try and go to the [Getting Started](https://nextjs.org/docs/getting-started) page in the documentation.

Then click the `create-next-app` link.

Is it just me or should I look at the issues first?

**Comment by jamesmosier** [comment, marked as answer]
Good catch!

Looks like this will be fixed in the next release: https://github.com/vercel/next.js/pull/16069 and thet original issue https://github.com/vercel/next.js/issues/16066

[Here is a link to the docs on GitHub](https://github.com/vercel/next.js/blob/canary/docs/api-reference/create-next-app.md)

**Reply by MathiasKandelborg** [reply]
That's what I ended up linking to the person needing it! Thanks for the references :smile: I didn't think to look, but also didn't want to make an issue if it was just my cache or whatever :sweat_smile:.
